src/common/settings_objects.cpp

```diff
@@ -4,39 +4,39 @@
 
 namespace PowerToysSettings {
 
-  Settings::Settings(const HINSTANCE hinstance, std::wstring_view powertoy_name) {
+  Settings::Settings(const HINSTANCE hinstance, const std::wstring_view &powertoy_name) {
     m_instance = hinstance;
     m_json.SetNamedValue(L"version", json::value(L"1.0"));
     m_json.SetNamedValue(L"name", json::value(powertoy_name));
     m_json.SetNamedValue(L"properties", json::JsonObject{});
   }
 
-  void Settings::set_description(UINT resource_id) {
+  void Settings::set_description( UINT resource_id) {
     m_json.SetNamedValue(L"description", json::value(get_resource(resource_id)));
   }
 
-  void Settings::set_description(std::wstring_view description) {
+  void Settings::set_description(const std::wstring_view &description) {
```

Comment:
```suggestion
  void Settings::set_description(const std::wstring_view& description) {
```
I think that is what @alekhyareddy28 had in mind.

Reply:
Yes. Thanks @davinci26.

Reply:
But set_description did not have pass by reff before, are you commenting from a commit?
![image](https://user-images.githubusercontent.com/58791731/74365441-f9304980-4d82-11ea-83cf-c271813699e0.png)